src/App.scss

```diff
@@ -535,9 +554,9 @@ model-viewer {
         margin-top: 10%;
         margin-bottom: 0%;
 
-        &__text {
-            font-size: 18px;
-            margin-left: -22%;  
+        .home__robot__text {
+            margin-left: -22%;
+            font-size: 15px;  
```

Comment:
You don't need to do .home__robot__text, it's already nested in that selector. Just add the margin-left and font-size options on lines 556 and 557.